apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -1,54 +1,649 @@
+import $ from 'jquery';
```

Comment:
This file looks crazy but what actually happened here is that I deleted the old TopInstructions and renamed the old TopInstructionsCSP to TopInstructions. So a lot of the "changes" here are actually existing code we had.

Reply:
If it is too hard to review it this way I can put the name back and then change the name in another PR.

Reply:
unfortunately, i think it might be too difficult to review with the file rename. do you mind reverting the rename so it's easier to see what actually changed in this file?